### DISCUSSION TITLE
Stay signed in?

### DISCUSSION THREAD

**Discussion by Roseghost** [alkem-io/alkemio#435] [Ideas] [answered]
I was wondering if the fact you got to log in every visit rather than staying auto logged in is done purposely? 

Every visit I have to log in again and I keep forgetting to do so untill I can't go further without logging in. I'm too used to the fact that other places keep you signed in. Maybe nice to implement?

**Comment by techsmyth** [comment, marked as answer]
Hi Rose, 

yes we had also noticed that, and made a mental note that it needed to be fixed. Now raised as a bug: https://github.com/alkem-io/client-web/issues/1160 + to be prioritized. Hopefully next sprint...

Cheers, Neil

ps. if you are happy that it is raised on the backlog properly then please also mark this discussion as answered...

**Reply by Roseghost** [reply]
Will do, Thanks! I'm not that familiar yet with the discussion part of github 😅
